### PR TITLE
Simplifying gtm pipeline

### PR DESCRIPTION
**Pull request by stuartio**
Removed AkamaiGtmTarget node type in favour of just using Endpoint directly.

### CODE REVIEW

**Comment by ccloes** [comment]
Lets split the AkamaiGtmTarget into these three types:
* An Endpoint node (if fqdn)
* An ipv4 node (if Ipv4 address)
* An ipv6 node (if Ipv6 address)

**Review by ccloes — CHANGES_REQUESTED** [review]
Lets change this per our discussion

File: nodestream_akamai/gtm/gtm.py [hunk 1]
```diff
@@ -10,35 +11,79 @@ def __init__(self, **akamai_client_kwargs) -> None:
...
-                    fqdn = property["name"] + "." + raw_domain["name"]
-                    servers = []
+            except Exception as err:
+                self.logger.error(f"Failed to get gtm domain: {domain['name']}")
+                self.logger.error(err)
+            deeplink_prefix = "https://control.akamai.com/apps/gtm/#/domains/"
+            deeplink = f'{deeplink_prefix}{domain["name"]}/properties/list'
+            properties = []
+            for property in raw_domain["properties"]:
+                fqdn = property["name"] + "." + raw_domain["name"]
```
Comment (jbristow) [inline nodestream_akamai/gtm/gtm.py:45]: Documenting my general request to double-check map gets to make sure that the null/missing case is handled.

Comment (stuart-macleod-akamai) [reply nodestream_akamai/gtm/gtm.py:52]: Good callout, but in this case these fields are always present in the API response. We could handle this, but it seems like overkill, and they are required so an error is not a terrible outcome if they are missing. Happy to accommodate, though

**Comment by stuart-macleod-akamai** [comment]
I've updated as per our discussions and the EDNS code. Should be simpler now

**Review by ccloes — APPROVED** [review]

**Pull request merged**